### PR TITLE
feat: more fields in CSV (issue 2325)

### PR DESCRIPTION
**Pull request by CharlesNepote**
### What

CSV refactoring

- documentation refactoring + .md file
- added:
  - ingredients_analysis_tags
  - additives_tags
  - nutrient_levels_tags
  - product_quantity
  - owner
  - data_quality_errors_tags
  - unique_scans_n
  - popularity_tags
  - completeness
  - last_image_t

### Related issue(s) and discussion
- Enhance #2325

### CODE REVIEW

File: html/data/data-fields.md [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+- `ingredients_analysis_tags`
+- `allergens`
+- `allergens_en`
+- `traces`
+- `traces_tags`
+- `traces_en`
+
+## misc. data
+
+- `serving_size` : serving size in g
```
Comment (stephanegigandet) [inline html/data/data-fields.md:95]:
```suggestion
- `serving_size` : text describing the serving size
```

File: html/data/data-fields.md [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+- `serving_quantity`
```
Comment (stephanegigandet) [inline html/data/data-fields.md:96]:
```suggestion
- `serving_quantity` : normalized serving size in grams or ml
```

**Review by stephanegigandet — APPROVED** [review]
I think we should fix the field names for nutrients that have a leading dash. It's a bug and it has no meaning.
What happened is that some fields had 1 or 2 leading dashes, and we just removed the first one. We should remove both.

File: html/data/data-fields.md [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+
+Please see https://wiki.openfoodfacts.org/Nutrients_handling_in_Open_Food_Facts for more information on nutrients.
+
+- `energy-kj_100g`
+- `energy-kcal_100g`
+- `energy_100g`
+- `energy-from-fat_100g`
+- `fat_100g`
+- `saturated-fat_100g`
+- `-butyric-acid_100g`
```
[the remark this answers is not in the document]
Comment (CharlesNepote) [reply html/data/data-fields.md:147]: This issue exists until 2017-09. I suggest we stay as we are not to break possible usages.

We can take more time later to build a completely new format with many more bug fixes and enhancements.

**Pull request merged**